src/main.rs

```diff
@@ -529,6 +532,54 @@ fn exist_audio() -> ExistResult {
     Ok(count > 0)
 }
 
+/// Determine whether audio is actively playing on any Pulseaudio interface.
+fn exist_audio_pulseaudio() -> ExistResult {
+    let users_output = Command::new("users")
+        .stderr(Stdio::null())
+        .output();
+    let users_stdout = users_output
+        .map_err(| _ | CircadianError("users failed".to_string()));
+    let users_output_str = String::from_utf8(users_stdout?.stdout)?;
+    let active_users: Vec<&str> = users_output_str.split(" ")
```

Comment:
How about changing `Vec` to `HashSet` to remove duplicates, or calling `active_users.dedup()`?  The `users` command can return the same user duplicated many times if you have a bunch of open shells.

Reply:
Thats a good idea. I did not know that it could return duplicates. I fixed it in the new commit.